### PR TITLE
apply bash style formatting for better readability and maintainability

### PR DESCRIPTION
**Pull request by girishmg**
@dcbw @danwinship PTAL

### CODE REVIEW

File: dist/images/ovnkube.sh [hunk 1]
```diff
@@ -352,54 +348,53 @@ process_healthy () {
...
 # $1 is the name of the process
 # $2 is the process pid
-check_health () {
+check_health() {
   ctl_file=""
   case ${1} in
-    "ovnkube"|"ovnkube-master")
-    ;;
-    "ovnnb_db"|"ovnsb_db")
+  "ovnkube" | "ovnkube-master") ;;
```
Comment (danwinship) [inline dist/images/ovnkube.sh:354]: This change seems weird to me. It makes more sense to me to always have the `;;` on its own line

File: dist/images/ovnkube.sh [hunk 2]
```diff
@@ -501,8 +495,8 @@ ovs-server () {
   while true; do
     if /usr/share/openvswitch/scripts/ovs-ctl status &>/dev/null; then
       echo "warning: Another process is currently managing OVS, waiting 10s ..." 2>&1
-      sleep 10 & wait
```
Comment (danwinship) [inline dist/images/ovnkube.sh:504]: That's weird... did you look through the history to see why it was that way? I wonder if there's some subtlety like in the case where kubelet kills the pod during the `sleep`.

File: dist/images/ovnkube.sh [hunk 3]
```diff
@@ -822,24 +812,24 @@ ovn-node () {
   fi
 
   OVN_ENCAP_IP=""
-  ovn_encap_ip=`ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip | tr -d '\"'`
+  ovn_encap_ip=$(ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip | tr -d '\"')
```
Comment (danwinship) [inline dist/images/ovnkube.sh:815]: the `tr` part is just implementing the equivalent of `--format=bare` isn't it?

File: dist/images/ovnkube.sh (hunk 2 above)
Comment (girishmg) [reply dist/images/ovnkube.sh:504]: @danwinship didn't find any history on this. I have added the wait back. Like you say, it helps in receiving signal whilst the script is sleeping in the background. Thanks.

File: dist/images/ovnkube.sh (hunk 3 above)
Comment (girishmg) [reply dist/images/ovnkube.sh:815]: If an user has provided the ovn-encap-ip with quotes, then --data=bare will not work. For example:

```
$ ovs-vsctl list open |grep external_ids
external_ids        : {hostname="sdn-test1", ovn-bridge-mappings="physnet:breno1", ovn-encap-ip="10.8.51.51", ovn-encap-type=geneve, ovn-nb="tcp:10.8.51.51:6641", ovn-openflow-probe-interval="180", ovn-remote="tcp:10.8.51.51:6642", ovn-remote-probe-interval="100000", rundir="/var/run/openvswitch", system-id="3b3e3241-2435-425d-bccf-d794465103dc"}
```

```
$ ovs-vsctl --data=bare get Open_vSwitch . external_ids:ovn-encap-ip
"10.8.51.51"
```

File: dist/images/ovnkube.sh (hunk 1 above)
Comment (girishmg) [reply dist/images/ovnkube.sh:354]: ok moved to its own line.

**Comment by dave-tucker** [comment]
I'm quite a fan of [shellcheck](https://github.com/koalaman/shellcheck) as it can also be integrated as a gate in CI. Having an automated check will make it easier for the improvements to remain in place and for new code to be of consistent style.

File: dist/images/ovnkube.sh (hunk 3 above)
Comment (girishmg) [reply dist/images/ovnkube.sh:817]: @danwinship PTAL. I have addressed your comments.

**Pull request merged**